### PR TITLE
CIV-3604: Try the new online service

### PR DESCRIPTION
**Pull request by hmcts-version1-pablo**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-3604

### Change description ###

Screen informing about the new online service.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/routes/urls.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const DIRECTIONS_QUESTIONNAIRE_URL = `${BASE_CASE_URL}/directions-questionnaire`
 const FULL_REJECTION_URL = `${BASE_CASE_RESPONSE_URL}/full-rejection`;
 const PARTIAL_ADMISSION_URL = `${BASE_CASE_RESPONSE_URL}/partial-admission`;
 const FULL_ADMISSION_URL = `${BASE_CASE_RESPONSE_URL}/full-admission`;
+export const ELIGIBILITY_URL = '/eligibility';
```
Comment (laperla11) [inline src/main/routes/urls.ts:9]: Can you pls change it to `const BASE_ELIGIBILITY_URL = '/eligibility';` to align with other eligibity PRs.

**Review by laperla11 — CHANGES_REQUESTED** [review]

File: src/main/routes/features/eligibility/tryNewServiceController.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as express from 'express';
+import {ELIGIBILITY_URL, ELIGIBILITY_CLAIM_VALUE_URL} from '../../urls';
+
+const tryNewServiceController = express.Router();
+const tryNewServiceViewPath = 'features/eligibility/try-new-service';
+
+tryNewServiceController.get(ELIGIBILITY_URL, async (_req, res) => {
```
Comment (GorilaNaranja) [inline src/main/routes/features/eligibility/tryNewServiceController.ts:7]: You can import express to set the types of req and res

`import * as express from 'express';`

`(req: express.Request, res: express.Response)`

File: src/main/routes/features/eligibility/tryNewServiceController.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  res.render(tryNewServiceViewPath);
+});
+
+tryNewServiceController.post(ELIGIBILITY_URL, async (req, res) => {
```
Comment (GorilaNaranja) [inline src/main/routes/features/eligibility/tryNewServiceController.ts:11]: same here, add types

File: src/main/views/features/eligibility/try-new-service.njk [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../../macro/page-title.njk" import setPageTitle %}
+
+{% block pageTitle %}
+  {{ setPageTitle('PAGES.TRY_NEW_SERVICE.PAGE_TITLE') }}
+{% endblock %}
+
+{% block content %}
+
+
```
Comment (GorilaNaranja) [inline src/main/views/features/eligibility/try-new-service.njk:13]: Not sure but I think sonar or lint complain about more than 1 empty line, please one empty line

**Review by GorilaNaranja — CHANGES_REQUESTED** [review]

File: src/main/views/features/eligibility/try-new-service.njk [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper" role="main">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-l">{{ t('PAGES.TRY_NEW_SERVICE.TITLE') }}</h1>
+
+          <p class="govuk-body">{{ t('PAGES.TRY_NEW_SERVICE.WE_ARE_BUILDING') }}</p>
+          <p class="govuk-body">{{ t('PAGES.TRY_NEW_SERVICE.YOU_WILL_BE_ASKED') }}</p>
+
+          <form method="post" novalidate>
```
[the remark this answers is not in the document]
Comment (hmcts-version1-pablo) [reply src/main/views/features/eligibility/try-new-service.njk:23]: Very good point! I'm going to modify it

File: src/main/routes/urls.ts (hunk 1 above)
Comment (hmcts-version1-pablo) [reply src/main/routes/urls.ts]: Done!

File: src/main/views/features/eligibility/try-new-service.njk (hunk 5 above)
[the remark this answers is not in the document]
Comment (hmcts-version1-pablo) [reply src/main/views/features/eligibility/try-new-service.njk]: Done!

File: src/main/routes/features/eligibility/tryNewServiceController.ts (hunk 2 above)
Comment (hmcts-version1-pablo) [reply src/main/routes/features/eligibility/tryNewServiceController.ts]: Done!

File: src/main/routes/features/eligibility/tryNewServiceController.ts (hunk 3 above)
Comment (hmcts-version1-pablo) [reply src/main/routes/features/eligibility/tryNewServiceController.ts]: Done!

**Review by GorilaNaranja — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Pull request merged**